### ISSUE TITLE
Unable to train using densenet 

### ISSUE THREAD

**Issue by KapoorHitesh** [AlexeyAB/darknet#862]
Hi ,

Could you help me understand what am I missing here?
I made the changes in the config file as mentioned here:
https://github.com/AlexeyAB/darknet/issues/179

I don't see any training happen. At the end I just see the below message once:

`
mask_scale: Using default '1.000000'
Loading weights from densenet201.weights...
 seen 32
Done!
Learning Rate: 0.0001, Momentum: 0.9, Decay: 0.0005
Saving weights to backup//densenet201_new_final.weights
`

What am I missing here? I am not getting any errors

command used to train
` ./darknet detector train cfg/obj.data cfg/densenet201_new.cfg densenet201.weights`

End of my .cfg file looks like below 
`[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[route]
layers=-1,-3

[convolutional]
size=1
stride=1
pad=1
filters=40
activation=linear

[region]
anchors =  1.3221, 1.73145, 3.19275, 4.00944, 5.05587, 8.09892, 9.47112, 4.84053, 11.2364, 10.0071
bias_match=1
classes=3
coords=4
num=5
softmax=1
jitter=.3
rescore=1

object_scale=5
noobject_scale=1
class_scale=1
coord_scale=1

absolute=1
thresh = .6
random=0
`
I also tried similar approach for resnet152 but facing similar situation

**Comment by KapoorHitesh** [comment]
I got the issue. I was using incorrect file.
